packages/select/src/vaadin-select.js

```diff
@@ -247,10 +247,8 @@ class Select extends OverlayClassMixin(
       renderer: Function,
 
       /**
-       * It stores the the `value` property of the selected item, providing the
-       * value for iron-form.
```

Comment:
We dropped `iron-form` support long ago, somehow this mention was left unnoticed.